static/app/views/preprod/utils.tsx

```diff
@@ -8,17 +8,15 @@ export function formatBytes(bytes: number, isSi = false) {
   return parseFloat((bytes / Math.pow(k, i)).toFixed(2)) + ' ' + sizes[i];
 }
 
-export function getPlatformFromArtifactType(
-  artifactType: BuildDetailsArtifactType
-): Platform {
-  switch (artifactType) {
-    case BuildDetailsArtifactType.XCARCHIVE:
-      return 'ios';
-    case BuildDetailsArtifactType.AAB:
-    case BuildDetailsArtifactType.APK:
+export function getPlatformIconFromPlatform(platform: Platform): string {
```

Comment:
nit: should this return a string literal union or something more specific than string?